js/parts-more/ColumnRangeSeries.js

```diff
@@ -138,17 +138,20 @@ seriesType('columnrange', 'arearange', merge(defaultPlotOptions.column, defaultP
     drawTracker: function () {
         return colProto.drawTracker.apply(this, arguments);
     },
-    findMinColumnWidth: function () {
-        return colProto.findMinColumnWidth.apply(this, arguments);
+    getMaxColumnCount: function () {
+        return colProto.getMaxColumnCount.apply(this, arguments);
+    },
+    getMinColumnWidth: function () {
+        return colProto.getMinColumnWidth.apply(this, arguments);
     },
```

Comment:
What is the reason of using function instead of `getMinColumnWidth: colProto.getMinColumnWidth` ?

Reply:
There is a difference between using `getMinColumnWidth: colProto.getMinColumnWidth` and function with `return colProto.getMinColumnWidth.apply(...)`.

- `getMinColumnWidth: colProto.getMinColumnWidth`
When we load core code and modules, then `columnrange.prototype.getMinColumnWidth` gets a value of `column.prototype.getMinColumnWidth`. Then, when we change (e.g. wrap) the `column.prototype.getMinColumnWidth`, the `columnrange`'s method is not changed.
- function with `return colProto.getMinColumnWidth.apply(...)`
Here, after we edited `getMinColumnWidth` on `column`'s prototype, the `columnrange` series will always fire the fresh, edited method.

Reply:
Makes sense and I agree.

Reply:
Could you add a comment with explanation there? Or is it somehow protected by a test, so no one will try to refactor this accidentally?

Reply:
Writing a test for each method would be overkill. On the other hand, writing comments at each method would be overkill too. What is the best approach then?

Reply:
I think one test to rule them all should be enough :) It would be nice to have all methods commented anyway